q2_types/ordination/tests/test_transformer.py

```diff
@@ -43,6 +45,40 @@ def test_ordination_format_to_skbio_ordination_results(self):
 
             self.assertEqual(str(exp), str(obs))
 
+    def test_1x1_ordination_format_to_metadata(self):
+        _, obs = self.transform_format(OrdinationFormat, qiime2.Metadata,
+                                       'pcoa-results-1x1.txt')
+
+        index = pd.Index(['s1'], name='Sample ID', dtype=object)
+        exp_df = pd.DataFrame([0.0], index=index, columns=['Axis 1'],
+                              dtype=float)
+        exp = qiime2.Metadata(exp_df)
+
+        self.assertEqual(exp, obs)
+
+    def test_2x2_ordination_format_to_metadata(self):
+        _, obs = self.transform_format(OrdinationFormat, qiime2.Metadata,
+                                       'pcoa-results-2x2.txt')
+
+        index = pd.Index(['s1', 's2'], name='Sample ID', dtype=object)
+        exp_df = pd.DataFrame([[-20.999999999999996, -0.0],
+                               [20.999999999999996, -0.0]], index=index,
+                              columns=['Axis 1', 'Axis 2'], dtype=float)
+        exp = qiime2.Metadata(exp_df)
+
+        self.assertEqual(exp, obs)
+
+    def test_NxN_ordination_format_to_metadata(self):
+        # Not creating a reference dataframe here because manually populating
+        # that DataFrame is a pain. Specifically we just want to check the
+        # functionality of the dynamic column naming (e.g. Axis N).
+        _, obs = self.transform_format(OrdinationFormat, qiime2.Metadata,
+                                       'pcoa-results-NxN.txt')
+        obs = obs.to_dataframe()
+
+        columns = ['Axis %d' % i for i in range(1, 9)]
+        self.assertEqual(columns, sorted(list(obs.columns.values)))
```

Comment:
Is this `sorted` call necessary? Also, I think you can use this API to get columns as a `list` object: `obs.columns.tolist()`